galaxy_ng/app/access_control/statements/pulp.py

```diff
@@ -577,6 +577,20 @@
             {"action": "*", "principal": "admin", "effect": "allow"},
         ]
     },
+    "tasks": {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["retrieve", "my_permissions"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+        ]
+    }
```

Comment:
@awcrosby do you have any insight as to why we need this now?

Reply:
Not sure... did you need to add this to get tests passing?